### PR TITLE
Implement function to fetch from /data and output to DOM (Servlet-2)

### PR DESCRIPTION
**Pull request by samuelsogbesan**
(Branched from servlet-1 branch, which is awaiting merge for some early commits. PR #16 )

- **Create JS function to fetch information from "/data"**
- **Connect button to function to output message to DOM**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -10,4 +10,15 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function printMessageFromServer(){
+  fetch("/data")
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/script.js]: [use single quotes for javascript string literals](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes), here and elsewhere

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -10,4 +10,15 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function printMessageFromServer(){
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/script.js]: [add a space between `)` and `{`](https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks)

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -10,4 +10,15 @@
...
+  fetch("/data")
+  .then(data => data.text())
+  .then(message => {
+    var container = document.getElementById("data-container");
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/script.js]: [don't use `var`](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -10,4 +10,15 @@
...
+    container.innerText = message;
+  })
+}
+
+window.onload = () => document.getElementById("data-button").addEventListener("click",printMessageFromServer);
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/script.js]: add a space between `,` and `printMessageFromServer`

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -11,7 +11,7 @@ body {
 }
 
 #fullpage {
-  min-width: 100vw;
+min-width: 100vw;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: indent by 2 spaces and maintain sorted declaration order

**Review by krasimirgg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Updated with commits df04000 and 6581af5

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/script.js]: Updated with commit 95f8b45

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/script.js]: Updated with commit 7a48eaa

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/script.js]: Updated in commit 0898ea8

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/script.js]: Updated with commit 95f8b45

**Review by krasimirgg — APPROVED** [review]
Thank you!

**Pull request merged**